frontend/connection/src/types.ts

```diff
@@ -193,6 +193,13 @@ export type LibConfig = {
 
   /** Deprecated. Use resourceCrossOriginMode instead. If set to true, the value of resourceCrossOriginMode will be "anonymous". */
   setAnonymousCrossOriginPropertyOnMediaElements?: boolean
+
+  /** When set to false, the download attribute will not be set on the link element. This is mainly for scenarios
+   * where a service worker is in the play, as the service worker will not intercept the download request otherwise
+   * (see https://issues.chromium.org/issues/40410035). The browser will still set the file name based on the server's
+   * Content-Disposition header.
+   */
+  setDownloadAttributeOnLinkElements?: boolean
```

Comment:
I think negating this might be a bit more consistent to follow the pattern to have false or not defined as the default behaviour, and only true to diverge from the default. E.g. preventDownloadAttribute, ignoreDownloadAttribute, ...

But maybe it's even a bit clearer to use something like usesDownloadServiceWorker, see comment below.